src/build.py

```diff
@@ -705,9 +724,26 @@ def CopyLLVMTools(build_dir, prefix=''):
       CopyLibraryToArchive(os.path.join(build_dir, 'lib', e), prefix)
 
 
+def WinBuildEnv(build_dir, use_gnuwin32=False, bin_subdir=False,
+                runtime='Release'):
+  if not IsWindows():
+    return None
+  cc_env = host_toolchains.SetUpVSEnv(build_dir)
+  if use_gnuwin32:
+    cc_env['PATH'] = cc_env['PATH'] + os.pathsep + os.path.join(GNUWIN32_DIR, 
```

Comment:
`os.pathsep` is `;` (on windows), i.e. the character that separates the directories in `%PATH%` (not to be confused with `os.sep` which is the slash :D )